### PR TITLE
Solution - https://github.com/NativeScript/docs/issues/393

### PR DESCRIPTION
**Pull request by bradmartin**

### CODE REVIEW

File: start/troubleshooting.md [hunk 1]
```diff
@@ -60,4 +60,13 @@ A problem occurred configuring root project 'app_name'.
...
\ No newline at end of file
+The NativeScript team is working on improving the performance of Android apps on initial run.
+
+### Added App_Resources Are Not Working
+
+**Problem:** When you add images or files to your app's *App_Resources* folder, run the app and the images don't show up when they should.
+
+**Solution:** When you add files to the *App_Resources* folder you have to run a new *build* of the app. Files located in *App_Resources* are compiled into the executable so they can't be synced with a previous build. Correct steps to take:
+  - Uninstall the old .apk (Android) or .ipa (iOS) from the device or emulator you are running the app on.
+  - Execute the `tns build android` or `tns build ios` command to create a new .apk or .ipa for your app.
```
Comment (tjvantoll) [inline start/troubleshooting.md:71]: Isn’t this step unnecessary if you’re going to do `tns run` right after this?

File: start/troubleshooting.md [hunk 2]
```diff
@@ -60,4 +60,13 @@ A problem occurred configuring root project 'app_name'.
 
 **Workaround:** None. Wait for the app to load.
 
-The NativeScript team is working on improving the performance of Android apps on initial run.
\ No newline at end of file
+The NativeScript team is working on improving the performance of Android apps on initial run.
+
+### Added App_Resources Are Not Working
```
Comment (tjvantoll) [inline start/troubleshooting.md:65]: How about “Files added to the App_Resources folder are not working”?

**Comment by tjvantoll** [comment]
Thanks @bradmartin! I just had a couple of trivial notes here.

Also just as an fyi, if you include “close”, “fix”, or “resolve” in your commit message GitHub will automatically associate this PR with the appropriate issue.

So you could do something like:

```
New troubleshooting tip for new files in App_Resources.

Fixes #393
```

**Comment by bradmartin** [comment]
Magic. Thanks. Yea I wasn't sure on the best wording for this. Also `run`
should produce the build correctly, I'm just not certain if it does a
complete build since you still have the executable in your directory. Might
be a good question for the CLI team. I just personally go the long route to
be extra careful. Do whatever you want with it, it's just come up a few
times recently so figured I'd try to get it documented.

**Comment by tjvantoll** [comment]
@bradmartin Will do. I’ll leave it as a precaution. I’ll change the heading and land this.

Thanks! 🎉

**Comment by tjvantoll** [comment]
Landed in f8539c8fedced0c8cf02a4016fe849cf89e10cbb. Also, welcome to the list of [NativeScript docs’ contributors](https://github.com/nativescript/docs#contributors)!

**Pull request closed without merging**